src/main/java/com/rtb/projectmanagementtool/project/ProjectController.java

```diff
@@ -26,60 +22,70 @@ public ProjectController(DatastoreService datastore) {
   }
 
   /**
-   * Gets all the projects in datastore that all specified users are in
+   * Gets all the projects in datastore that all users are in
    *
    * @param userIds the collection of userIds whose projects are to be retrieved
    * @return HashSet containing desired projects
    */
-  public HashSet<ProjectData> getProjects(Collection userIds) {
-
-    HashSet<ProjectData> projectContainer = new HashSet<ProjectData>();
+  public ArrayList<ProjectData> getAllProjects() {
+    return getProjects(0l, false);
+  }
 
-    PreparedQuery results = datastore.prepare(new Query("Project"));
-    for (Entity entity : results.asIterable()) {
-      if (entityContainsUser(entity, userIds)) {
-        ProjectData project = new ProjectData(entity);
-        projectContainer.add(project);
-      }
-    }
+  /**
+   * Gets all the projects in datastore that the specified user is in
+   *
+   * @param userId the id of user whose projects to retrieve
+   * @return HashSet containing desired projects
+   */
+  public ArrayList<ProjectData> getUserProjects(long userId) {
+    return getProjects(userId, false);
+  }
 
-    return projectContainer;
+  /**
+   * Gets all the projects in datastore created by user
+   *
+   * @param userId the id of owner whose projects to retrieve
+   * @return HashSet containing desired projects
+   */
+  public ArrayList<ProjectData> getCreatorProjects(long userId) {
```

Comment:
nit:  `Creator` in this form is an adjective of `Project`, whereas the intention here is for `Creator` to describe the type of user.  Maybe `getProjectsCreatedByUser` or `getProjectsWithCreator`